x-pack/plugins/apm/public/components/shared/PropertiesTable/NestedKeyValueTable.tsx

```diff
@@ -56,24 +56,30 @@ const EmptyValue = styled.span`
   color: ${colors.gray3};
 `;
 
-function formatValue(value: any): JSX.Element {
+export function FormattedKey({
+  k,
+  value
+}: {
+  k: string;
+  value: any;
+}): JSX.Element {
+  if (value == null) {
+    return <EmptyValue>{k}</EmptyValue>;
+  }
+
+  return <React.Fragment>{k}</React.Fragment>;
+}
```

Comment:
I think this is a bit harder on the eyes compared to `formatValue`.

Reply:
(Harder on the eyes, mostly due to the function signature, where the object param makes types more verbose)